digicampipe/tests/test_lid_ccd.py

```diff
@@ -0,0 +1,54 @@
+from digicampipe.image.sky_image import *
+from digicampipe.image.cones_image import *
+
+from pkg_resources import resource_filename
+from glob import glob
+
+example_lid_CCD_image_file_paths = glob(
+    resource_filename('digicampipe', 'tests/resources/stars_on_lid/*.fits')
+)
+
+
+def test_find_stars():
+    # find stars in lid CCD images:
+    crop_pixels1 = [  # (850, 50),
+        # (550, 500),
+        (350, 900),
+        # (600, 1350),
+        (850, 1800), ]
+    crop_pixels2 = [  # (1100, 650),
+        # (900, 1000),
+        (770, 1550),
+        # (1000, 2050),
+        (1300, 2400), ]
```

Comment:
a rectangle is defined by the the combination of an element in crop_pixels1 and an element in crop_pixels2, which share the same index. So one could say:

    rectangles = list(zip(crop_pixels1, crop_pixels2))

I think currently the interface is very easy to misunderstand or misuse, one might by accident give a too long list as `crop_pixel1` and so on.

I assume it would be easier to invent a small class `Rectangle`, e.g. like this:

    class Rectangle:
        def __init__(self, bottom, left, top, right):
            # ...

And give `LidCCDObservation` a list of rectangles.

Reply:
That would totally make sense

Reply:
fixed in 4724d7158293